task.env

```diff
@@ -0,0 +1 @@
+PYRIGHT_PYTHON_FORCE_VERSION=1.1.221  # should match .github/workflows/lint.yml
```

Comment:
this comment is no longer accurate, this is used with dotenv in the github workflow